base/statistics.jl

```diff
@@ -605,10 +605,11 @@ median{T}(v::AbstractArray{T}, region) = mapslices(median!, v, region)
 """
     quantile!([q, ] v, p; sorted=false)
 
-Compute the quantile(s) of a vector `v` at the probabilities `p`, with optional output into
-array `q` (if not provided, a new output array is created). The keyword argument `sorted`
-indicates whether `v` can be assumed to be sorted; if `false` (the default), then the
-elements of `v` may be partially sorted.
+Compute the quantile(s) of a vector `v` at the probability or vector or tuple of
```

Comment:
"probability or vector or tuple" sounds kind of confusing IMO. Would something like "at the probability or probabilities `p`, given as a single value, a vector, or a tuple" be reasonable?